### ISSUE TITLE
zsh: too many levels of symbolic links: kumactl

### ISSUE THREAD

**Issue by jaikratsinghtariyal** [kumahq/kuma#2774]
### Summary

Facing error on terminal while hitting command `zsh: too many levels of symbolic links: kumactl`

### Steps To Reproduce

1. echo "type: Mesh name: my-first-mesh" | kumactl apply -f -
.

### Additional Details & Logs

Following [this](https://kuma.io/docs/1.3.0/installation/macos/)

- Version 
- Error logs -- zsh: too many levels of symbolic links: kumactl
- Configuration
- Platform and Operating System -- MAC OS
- Installation Method (Helm, kumactl, AWS CloudFormation, etc.) -- kumactl

**Comment by bartsmykla** [comment]
Hi @jaikratsinghtariyal, thank you for raising issue, but could you give us more information about how to reproduce this issue? What command you are trying to run?

**Comment by jaikratsinghtariyal** [comment]
Command is I putting is `echo "type: Mesh name: my-first-mesh" | kumactl apply -f -`

I am setting up kuma first time on my machine by following link https://kuma.io/docs/1.3.0/installation/macos/

**Comment by jaikratsinghtariyal** [comment]
Also... after spending time this way when I tried installing the same using docker getting below error..

**Command**- 
docker run \
    -p 5681:5681 \
    docker.io/kumahq/kuma-cp: run

**Error** - 
docker: invalid reference format.
See 'docker run --help'.

**Comment by bartsmykla** [comment]
The value which you provide in `echo "..."` has to be a valid yaml, which would be valid if you would have new line before `name: my-first-mesh`.

Could you try:
```sh
echo "type: Mesh
name: my-first-mesh" | kumactl apply -f -
```

**Comment by jaikratsinghtariyal** [comment]
![image](https://user-images.githubusercontent.com/40471286/133250723-3a659d6c-5f80-4702-a7bd-9a32120a2c51.png)

I copied the command from Kuma Portal running on my machine. Attaching image.

even command `echo -e "type: Mesh\nname: my-first-mesh" | kumactl apply -f -` is also giving the same error.
i.e. `zsh: too many levels of symbolic links: kumactl`

**Comment by bartsmykla** [comment]
@jaikratsinghtariyal this is the correct docker command you should use for docker:
```sh
docker run -p 5681:5681 docker.io/kumahq/kuma-cp:1.3.0 run
```

**Comment by jaikratsinghtariyal** [comment]
Yes, this docker command is working. Then please update documentation on docker at https://kuma.io/docs/1.3.0/installation/docker/

![image](https://user-images.githubusercontent.com/40471286/133251338-5f60585f-e180-473d-a0f3-037f45eb14a7.png)

But I think this is half work done. Somehow kubectl command is not working.

**Comment by bartsmykla** [comment]
> ![image](https://user-images.githubusercontent.com/40471286/133250723-3a659d6c-5f80-4702-a7bd-9a32120a2c51.png)
> 
> I copied the command from Kuma Portal running on my machine. Attaching image.
> 
> even command `echo -e "type: Mesh\nname: my-first-mesh" | kumactl apply -f -` is also giving the same error.
> i.e. `zsh: too many levels of symbolic links: kumactl`

It looks like the problem lies on your OS side, how have you installed kumactl?

**Comment by jaikratsinghtariyal** [comment]
curl -L https://kuma.io/installer.sh | sh -

like its mentioned at https://kuma.io/docs/1.3.0/installation/macos/

**Comment by bartsmykla** [comment]
@jaikratsinghtariyal thank you for pointing this out. Could you create an issue with this docs problems here: https://github.com/kumahq/kuma-website/?

**Comment by bartsmykla** [comment]
> curl -L https://kuma.io/installer.sh | sh -
> 
> like its mentioned at https://kuma.io/docs/1.3.0/installation/macos/

Just looked into the script https://kuma.io/installer.sh and there is no linking, how have you placed `kumactl` in the `PATH`?

**Comment by jaikratsinghtariyal** [comment]
Yes, kumactl is working...otherwise I would get kubctl command not found something.

**Comment by bartsmykla** [comment]
sorry, but I don't understand your answer in context of my question

**Comment by jaikratsinghtariyal** [comment]
I mean, yes I set it in PATH like below.

`export PATH=/Users/jaikratsinghtariyal/Sandbox/kuma-1.3.0/bin:$PATH`

Is it correct way?

**Comment by bartsmykla** [comment]
Is any directory in this path `/Users/jaikratsinghtariyal/Sandbox/kuma-1.3.0/` a symbolic link?

**Comment by jaikratsinghtariyal** [comment]
![image](https://user-images.githubusercontent.com/40471286/133254448-a0d976c4-b98e-4d33-a841-9316931c9ba7.png)

Question is not clear but this is what I see inside `/kuma-1.3.0`

**Comment by bartsmykla** [comment]
Can you call `file /Users/jaikratsinghtariyal/Sandbox/kuma-1.3.0/bin/kumactl`?

**Comment by jaikratsinghtariyal** [comment]
Yes, its working...even command `echo "type: Mesh                                          
name: my-first-mesh" | /Users/jaikratsinghtariyal/Sandbox/kuma-1.3.0/bin/kumactl apply -f -` started working.

**Comment by lahabana** [comment]
This seems to be a problem with symlinks on the host machine so closing this.
